noxfile.py

```diff
@@ -86,10 +86,11 @@ def install_ci(session, group):
     if group == "unit-core":
         extras += "bionty,aws,gcp,zarr,fcs,jupyter"
         run(session, "uv pip install --system huggingface_hub")
+        run(
+            session, "uv pip install --system tiledbsoma==1.15.0rc3"
```

Comment:
Please add a note why you're pinning this version.

Reply:
Added.